discovery_engine_core/core/src/ranker.rs

```diff
@@ -97,6 +100,10 @@ impl Ranker for xayn_ai::ranker::Ranker {
     fn negative_cois(&self) -> &[NegativeCoi] {
         self.negative_cois()
     }
+
+    fn remove_key_phrases(&mut self, markets: &[Market]) {
+        self.remove_key_phrases(markets)
```

Comment:
🔧 `just check` locally complains about this, not sure why the ci doesn't
```suggestion
        self.remove_key_phrases(markets);
```